mlir/utils/miopenParameterSweeps.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+"""Script to sweep the parameters of the MIOpen driver for bugs
```

Comment:
So this script is to allow the user to define PARAM_ITERATORS for testing random configurations? Do we plan to run it nightly? 

You may want to mention that the script is meant to run on gfx908 unless the user manually updates the xdlops field of PARAM_ITERATORS.

Reply:
I figured that, if this even gets run automatically, it'll be a weekly job